src/components/Pagination/_Pagination.scss

```diff
@@ -11,13 +11,32 @@
         }
     }
 
-    .label {
-        @include space-stack-none;
+    &__list {
+        @include list-reset;
 
-        align-self: center;
+        display: flex;
     }
 
-    .select {
-        @include space-stack-none;
+    &__item {
+        list-style-type: none;
+    }
+
+    &__link {
+        line-height: 1.15;
+        padding: 8px;
+        text-decoration: none;
+
+        &.selected {
+            color: var(--ui-black);
+        }
+
+        &[role="button"] {
```

Comment:
Perhaps this should be moved into `_Link.scss`, since it seems like something you'd want to happen universally.